src/features/evaluations/[id]/EvaluationsCriteria.tsx

```diff
@@ -0,0 +1,216 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import { Button } from "../../../components/button/Button"
+import { StarRating } from "../../../components/rating/StarRating"
+import {
+  getEvaluationTemplateContents,
+  updateEvaluationRatingById,
+} from "../../../redux/slices/evaluationTemplateContentsSlice"
+import { useAppDispatch } from "../../../hooks/useAppDispatch"
+import { useAppSelector } from "../../../hooks/useAppSelector"
+import {
+  submitAnswer,
+  submitComment,
+  submitEvaluation,
+  updateEvaluationStatusById,
+} from "../../../redux/slices/userSlice"
+import { Loading } from "../../../types/loadingType"
+import { setAlert } from "../../../redux/slices/appSlice"
+import {
+  EvaluationStatus,
+  type Evaluation,
+} from "../../../types/evaluationType"
+import { formatDate } from "../../../utils/formatDate"
+import { TextArea } from "../../../components/textarea/TextArea"
+
+export const EvaluationsCriteria = () => {
+  const { id, evaluation_id } = useParams()
+  const appDispatch = useAppDispatch()
+  const { evaluation_template_contents } = useAppSelector(
+    (state) => state.evaluationTemplateContents
+  )
+  const { loading, loading_comment, loading_answer, user_evaluations } =
+    useAppSelector((state) => state.user)
+  const [evaluation, setEvaluation] = useState<Evaluation>()
+  const [comment, setComment] = useState<string>("")
+  const [errorMessage, setErrorMessage] = useState<string>("")
```

Comment:
use `string | null` 🤔

Reply:
![image](https://github.com/nerubia/kh360-react/assets/91468846/925e273e-afc4-415f-ab3c-3e51dddc1fd3)
Should I use string | undefined instead kuys?

Reply:
![image](https://github.com/nerubia/kh360-react/assets/91468846/d3d11ecb-a824-457d-9c73-1e8fbceef24d)
Or change the TextAreaInterface types for value and error like this?

Reply:
keep it for now, or make separate PR to update `Input` and `TextArea` 🤔

Reply:
ayy joke do you mean only this line `const [errorMessage, setErrorMessage] = useState<string>("")` kuys ang i- string | null?

Reply:
yes, and use `null` as default value, so we don't need extra checking on `TextArea`

Reply:
alright, I don't have to change pala TextAreaInterface